Cabal-syntax/src/Distribution/Types/LegacyExeDependency.hs

```diff
@@ -18,7 +18,7 @@ import qualified Text.PrettyPrint                as Disp
 --
 -- It is "legacy" because we do not know what the build-tool referred to. It
 -- could refer to a pkg-config executable (PkgconfigName), or an internal
--- executable (UnqualComponentName). Thus the name is stringly typed.
```

Comment:
This is probably meant to be as is (`stringly`). It's a sort of pun.

Reply:
Oh, I checked again and now "stringly" makes sense to me in this context. Then it was wrong fix.

Reply:
Should I make commit that reverts this change?

Reply:
"strongly" is definitely wrong. It's either "weakly" typed or you may just put "strongly" back. I'm leaning towards putting it back but I don't feel strongly 😉 about it.

Reply:
(sorry for the typo: put "stringly" back, of course)